src/pages/Profile/index.js

```diff
@@ -23,90 +32,361 @@ import avatar from '../../assets/avatar.svg';
+                              <Button
+                                variant="secondary"
+                                className={styles.profileBtn}
+                                onClick={this.cancleEdit}
+                              >
+                                Cancel{' '}
+                              </Button>
+                            </GridItem>
+                          </Grid>
+                        )}
+                      </Grid>
+                    </div>
+                  </Card>
+                </GridItem>
+                <GridItem span={3}>
+                  <div>
+                    <Card className={styles.card}>
+                      <div className={styles.subHeader}>
+                        <KeyIcon />
+                        <span className={styles.subHeader}>Settings</span>
+                        <Grid className={styles.topCard}>
+                          <GridItem span={12} className={styles.subCardDiv}>
+                            <TextContent>
+                              <span>Account creation Date</span>
+                              <Text component={TextVariants.h4} style={{ marginTop: '0px' }}>
+                                (dd/mm/yyyy)
+                              </Text>
+                            </TextContent>
+                          </GridItem>
+                          <GridItem span={12} className={styles.subCardDiv}>
+                            <TextContent>
+                              <span>Data Expiration Limit</span>
+                            </TextContent>
+                            <FormSelect
+                              value={dataExpireValue}
+                              onChange={this.ondataExpireValueChange}
+                              aria-label="FormSelect Input"
+                              className={styles.expireSelect}
+                            >
+                              {this.dataExpireOptions.map(option => (
+                                // eslint-disable-next-line react/no-array-index-key
+                                <FormSelectOption
+                                  isDisabled={option.disabled}
+                                  value={option.value}
+                                  label={option.label}
+                                />
+                              ))}
+                            </FormSelect>
+                          </GridItem>
+                        </Grid>
+                        <Grid>
+                          <GridItem span={12} className={styles.subCardDiv}>
+                            <Button variant="link" icon={<RedoIcon />}>
+                              Reset All Data
+                            </Button>{' '}
+                          </GridItem>
+                          <GridItem span={12} className={styles.subCardDiv}>
+                            <TextContent>
+                              <Text component={TextVariants.h5}>No of results expiring</Text>
```

Comment:
I'd rather see "Number" than "No" ... Or maybe just reverse it and have "123 results expiring".

The hardcoded "123" seems odd here, too, though at this point it'd be hardcoded somewhere, like in a reducer.

Reply:
Fixed! Added to mock file.